Installation issue: Darshan-runtime GCC 9.3.0 Ubuntu 20.04

Using `spack@develop` with commit id `70c3b0ba0941` from 3-June-2020.

System:
* Ubuntu 20.04
* GCC 9.3.0
* x86_64 and ppc64le

```
$> spack install darshan-runtime
...
==> Error: Failed to install darshan-runtime due to ChildError: ProcessError: Command exited with status 2:
    'make' '-j16'
1 error found in build log:
     99     ar rcs lib/libdarshan-stubs.a lib/darshan-pnetcdf-stubs.o
     100    ../darshan-runtime/lib/darshan-core.c: In function 'darshan_get_exe_and_mounts':
     101    ../darshan-runtime/lib/darshan-core.c:1150:13: warning: ignoring return value of 'fgets', declared with attribute warn_unused_result [-Wunused-re
            sult]
     102     1150 |             fgets(cmdl,DARSHAN_EXE_LEN,fh);
     103          |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     104    /tmp/ccpXnisF.s: Assembler messages:
  >> 105    /tmp/ccpXnisF.s:8281: Error: symbol `__isoc99_fscanf' is already defined
     106    make: *** [Makefile:143: lib/darshan-stdio.po] Error 1
     107    make: *** Waiting for unfinished jobs....

See build log for details:
  /tmp/root/spack-stage/spack-stage-darshan-runtime-3.1.8-75i6acapt7rptrmgkibmuizjlciagw4a/spack-build-out.txt
```

Full build log: [darshan-runtime.txt](https://github.com/spack/spack/files/4724615/darshan-runtime.txt)

@carns  @shanedsnyder @michaelkuhn @carsonwoods

--- a/var/spack/repos/builtin/packages/darshan-runtime/package.py
+++ b/var/spack/repos/builtin/packages/darshan-runtime/package.py
@@ -21,6 +21,9 @@ class DarshanRuntime(Package):
     maintainers = ['shanedsnyder', 'carns']
 
     version('develop', branch='master')
+    version('3.2.1', sha256='d63048b7a3d1c4de939875943e3e7a2468a9034fcb68585edbc87f57f622e7f7')
+    version('3.2.0-pre1', sha256='afc11609420bab99a5f2dd7b4f4aa4c34f5423a5ca438c5f2d444dc3efe6dfa8')
+    version('3.2.0', sha256='4035435bdc0fa2a678247fbf8d5a31dfeb3a133baf06577786b1fe8d00a31b7e')
     version('3.1.8', sha256='3ed51c8d5d93b4a8cbb7d53d13052140a9dffe0bc1a3e1ebfc44a36a184b5c82')
     version('3.1.7', sha256='9ba535df292727ac1e8025bdf2dc42942715205cad8319d925723fd88709e8d6')
     version('3.1.6', sha256='21cb24e2a971c45e04476e00441b7fbea63d2afa727a5cf8b7a4a9d9004dd856')

--- a/var/spack/repos/builtin/packages/darshan-runtime/package.py
+++ b/var/spack/repos/builtin/packages/darshan-runtime/package.py
@@ -22,7 +22,6 @@ class DarshanRuntime(Package):
 
     version('develop', branch='master')
     version('3.2.1', sha256='d63048b7a3d1c4de939875943e3e7a2468a9034fcb68585edbc87f57f622e7f7')
-    version('3.2.0-pre1', sha256='afc11609420bab99a5f2dd7b4f4aa4c34f5423a5ca438c5f2d444dc3efe6dfa8')
     version('3.2.0', sha256='4035435bdc0fa2a678247fbf8d5a31dfeb3a133baf06577786b1fe8d00a31b7e')
     version('3.1.8', sha256='3ed51c8d5d93b4a8cbb7d53d13052140a9dffe0bc1a3e1ebfc44a36a184b5c82')
     version('3.1.7', sha256='9ba535df292727ac1e8025bdf2dc42942715205cad8319d925723fd88709e8d6')

--- a/var/spack/repos/builtin/packages/darshan-util/package.py
+++ b/var/spack/repos/builtin/packages/darshan-util/package.py
@@ -20,7 +20,6 @@ class DarshanUtil(Package):
 
     version('develop', branch='master')
     version('3.2.1', sha256='d63048b7a3d1c4de939875943e3e7a2468a9034fcb68585edbc87f57f622e7f7')
-    version('3.2.0-pre1', sha256='afc11609420bab99a5f2dd7b4f4aa4c34f5423a5ca438c5f2d444dc3efe6dfa8')
     version('3.2.0', sha256='4035435bdc0fa2a678247fbf8d5a31dfeb3a133baf06577786b1fe8d00a31b7e')
     version('3.1.8', sha256='3ed51c8d5d93b4a8cbb7d53d13052140a9dffe0bc1a3e1ebfc44a36a184b5c82')
     version('3.1.7', sha256='9ba535df292727ac1e8025bdf2dc42942715205cad8319d925723fd88709e8d6')

--- a/var/spack/repos/builtin/packages/darshan-util/package.py
+++ b/var/spack/repos/builtin/packages/darshan-util/package.py
@@ -19,6 +19,9 @@ class DarshanUtil(Package):
     maintainers = ['shanedsnyder', 'carns']
 
     version('develop', branch='master')
+    version('3.2.1', sha256='d63048b7a3d1c4de939875943e3e7a2468a9034fcb68585edbc87f57f622e7f7')
+    version('3.2.0-pre1', sha256='afc11609420bab99a5f2dd7b4f4aa4c34f5423a5ca438c5f2d444dc3efe6dfa8')
+    version('3.2.0', sha256='4035435bdc0fa2a678247fbf8d5a31dfeb3a133baf06577786b1fe8d00a31b7e')
     version('3.1.8', sha256='3ed51c8d5d93b4a8cbb7d53d13052140a9dffe0bc1a3e1ebfc44a36a184b5c82')
     version('3.1.7', sha256='9ba535df292727ac1e8025bdf2dc42942715205cad8319d925723fd88709e8d6')
     version('3.1.6', sha256='21cb24e2a971c45e04476e00441b7fbea63d2afa727a5cf8b7a4a9d9004dd856')

